<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTableWidget" name="view">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="autoScrollMargin">
       <number>16</number>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>12</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>22</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>30</number>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>记忆量</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>创建时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>说明</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="data">
    <property name="title">
     <string>数据</string>
    </property>
    <addaction name="item_add"/>
    <addaction name="item_remove"/>
   </widget>
   <widget class="QMenu" name="find">
    <property name="title">
     <string>查找</string>
    </property>
    <addaction name="search"/>
    <addaction name="screen"/>
    <addaction name="sort"/>
   </widget>
   <widget class="QMenu" name="edit">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="content"/>
    <addaction name="item_change"/>
   </widget>
   <addaction name="data"/>
   <addaction name="find"/>
   <addaction name="edit"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>记忆量低于...</string>
   </property>
  </action>
  <action name="info">
   <property name="text">
    <string>详细内容</string>
   </property>
  </action>
  <action name="concent">
   <property name="text">
    <string>项目信息</string>
   </property>
  </action>
  <action name="item_add">
   <property name="text">
    <string>添加项</string>
   </property>
   <property name="toolTip">
    <string>add</string>
   </property>
  </action>
  <action name="item_remove">
   <property name="text">
    <string>删除项</string>
   </property>
   <property name="toolTip">
    <string>remove</string>
   </property>
  </action>
  <action name="query">
   <property name="text">
    <string>查找项</string>
   </property>
  </action>
  <action name="search">
   <property name="text">
    <string>搜索</string>
   </property>
   <property name="toolTip">
    <string>search</string>
   </property>
  </action>
  <action name="screen">
   <property name="text">
    <string>筛选</string>
   </property>
   <property name="toolTip">
    <string>screen</string>
   </property>
  </action>
  <action name="sort">
   <property name="text">
    <string>排序</string>
   </property>
   <property name="toolTip">
    <string>sort</string>
   </property>
  </action>
  <action name="content">
   <property name="text">
    <string>内容</string>
   </property>
   <property name="toolTip">
    <string>content</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>信息</string>
   </property>
   <property name="toolTip">
    <string>info</string>
   </property>
  </action>
  <action name="item_change">
   <property name="text">
    <string>信息</string>
   </property>
   <property name="toolTip">
    <string>item_change</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
